documentation/schema.yaml

```diff
@@ -320,6 +320,22 @@ referencedSchemas:
         type: string
     discriminator:
       propertyName: type
+  Broadcast:
```

Comment:
MovingPicture was not added to the previous PR because the test code was not activated